lib/monitoring/aws-ecs-patterns/Ec2ServiceMonitoring.ts

```diff
@@ -102,6 +102,19 @@ interface BaseLoadBalancedEc2ServiceMonitoringProps
     string,
     MinProcessedBytesThreshold
   >;
+
+  /**
+   * Invert the statistics of `HealthyHostCount` and `UnHealthyHostCount`.
+   *
+   * When `invertLoadBalancerStatisticsOfTaskCountEnabled` is set to false, the minimum of `HealthyHostCount` and the maximum of `UnHealthyHostCount` are monitored.
+   * When `invertLoadBalancerStatisticsOfTaskCountEnabled` is set to true, the maximum of `HealthyHostCount` and the minimum of `UnHealthyHostCount` are monitored.
+   *
+   * `invertLoadBalancerStatisticsOfTaskCountEnabled` is recommended to set to true as per the guidelines at
+https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-cloudwatch-metrics.html#metric-statistics
+   *
+   * @default false
+   */
+  readonly invertLoadBalancerStatisticsOfTaskCountEnabled?: boolean;
```

Comment:
Maybe `invertLoadBalancerTaskCountMetricsStatistics`?

Reply:
Thank you for your review of my pull request, and I agree that `invertLoadBalancerTaskCountMetricsStatistics` is more appropriate. I will make the necessary changes and update the pull request as soon as possible. Thank you again for your feedback and guidance.